ra-optimisation/search-tree-rao/src/main/java/com/farao_community/farao/search_tree_rao/castor/algorithm/AutomatonSimulator.java

```diff
@@ -486,16 +557,22 @@ Pair<PrePerimeterResult, Map<RangeAction<?>, Double>> shiftRangeActionsUntilFlow
                 return Pair.of(automatonRangeActionOptimizationSensitivityAnalysisOutput, activatedRangeActionsWithSetpoint);
```

Comment:
we should probably replace the condition with "abs(direction) > 1e-6" to make sure we don't observe random problems

Reply:
ok